test/e2e/apimachinery/namespace.go

```diff
@@ -49,7 +49,7 @@ func extinguish(f *framework.Framework, totalNS int, maxAllowedAfterDel int, max
 			defer wg.Done()
 			defer ginkgo.GinkgoRecover()
 			ns := fmt.Sprintf("nslifetest-%v", n)
-			_, err = f.CreateNamespace(ns, nil)
+			_, err := f.CreateNamespace(ns, nil)
```

Comment:
remove `var err error` above?

Reply:
Thanks for pointing out. Code changes done.